spec/views/teachers/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
please remove `spec/views/` folder at all, we don't write view specs

Reply:
removed